Source/Core/DolphinQt/Settings/GameCubePane.cpp

```diff
@@ -186,12 +189,28 @@ void GameCubePane::OnConfigPressed(int slot)
   {
     bool ok;
     const auto new_mac = QInputDialog::getText(
-        this, tr("Broadband Adapter MAC address"), tr("Enter new Broadband Adapter MAC address:"),
-        QLineEdit::Normal, QString::fromStdString(SConfig::GetInstance().m_bba_mac), &ok);
+        // i18n: TAP is a type of virtual network interface, MAC stands for Media Access Control
+        // A MAC address uniquely identifies a network interface (physical) like a serial
+        // number. "TAP" and "MAC" should be kept in translations.
+        this, tr("Broadband Adapter (TAP) MAC address"),
```

Comment:
The MAC is still used in both cases (TAP and XLink Kai), should this translation string really be updated to include the word "TAP"?

Also, since XLink Kai _also_ requires a MAC (not to mention from a specific vendor; as validated by the `CEXIETHERNET` ctor), it appears that currently you have to select TAP to generate (or update) the MAC and then switch to XLink Kai to update the Destination Address.
Is this something we want to address here (assuming I'm not mistaken by just reading the code; I didn't run or test anything)?

Reply:
I've updated the wording used